packages/handlers/grpc/src/index.ts

```diff
@@ -157,7 +174,14 @@ export default class GrpcHandler implements MeshHandler {
             if (name !== this.config.serviceName) {
               rootFieldName = camelCase(name + '_' + rootFieldName);
             }
-            if (currentPath !== this.config.packageName) {
+            if (this.config.useReflection) {
+              const reflectionPath = currentPath.replace(this.config.serviceName || '', '');
+              rootFieldName = camelCase(currentPath.split('.').join('_') + '_' + rootFieldName);
+              responseType = camelCase(
+                currentPath.split('.').join('_') + '_' + responseType.replace(reflectionPath, '')
+              );
+              requestType = camelCase(currentPath.split('.').join('_') + '_' + requestType.replace(reflectionPath, ''));
```

Comment:
This stuff was tricky. The `requestType` and `responseType` properties under `methods` are in different format when we use reflection.